app/globals.css

```
@@ -99,24 +100,56 @@ button:focus-visible, a:focus-visible { outline: 2px solid var(--ink); outline-o
+.material-file:hover .material-file__context-control--set-aside,
+.material-file__context-control--set-aside:focus-visible { opacity: .68; pointer-events: auto; }
+.material-file:hover .material-file__context-control--set-aside svg,
+.material-file__context-control--set-aside:focus-visible svg { transform: rotate(0deg) scale(1); }
+.material-file__context-control--set-aside:focus-visible { opacity: 1; }
+.material-file__context-control--restore { width: var(--material-file-action-width); color: var(--index-muted); opacity: .86; }
 .material-file__context-control:disabled { cursor: default; opacity: .22; }
-.material-file:hover .material-file__context-control,
-.material-file__context-control:focus-visible { opacity: 1; }
 .material-file[data-context-excluded="true"] { color: color-mix(in srgb, var(--index-ink) 44%, transparent); }
-.material-file[data-context-excluded="true"] .material-file__context-control { opacity: .74; }
+.material-file[data-context-excluded="true"] .material-file__title { color: currentColor; }
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)